### PR TITLE
Linter learning first pull request

### PR DESCRIPTION
**Pull request by AlphaNtihinduka**
The repository is focused on the correct use of linter in Microverse Day 2 learning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
-# Hello-world
\ No newline at end of file
+# HTML & CSS3 Course
```
Comment (gemmen29) [inline README.md]: Kindly use the Microverse README template for the README file. and update it with your information. you can find it in this [link](https://github.com/microverseinc/readme-template)

File: README.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+
+If you are not familiar with linters, read [root level README](../README.md).
```
Comment (gemmen29) [inline README.md]: Kindly add a description for your PR with what you did exactly in this PR in order for your PR to be professional.

File: README.md [hunk 3]
```diff
@@ -1 +1,97 @@
...
+
```
Comment (gemmen29) [inline README.md:3]: [OPTIONAL] Kindly rename the branch with a meaningful name, not `feature1`.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @AlphaNtihinduka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AlphaNtihinduka) [reply README.md]: @gemmen29 I have made the changes and updated my repository to meet the professional standards. I would kindly request support on the submission of the changes made. Thank you.

**Pull request closed without merging**